test/Rings/mpolyquo-localizations.jl

```diff
@@ -30,7 +30,7 @@
   W = MPolyQuoLocalizedRing(S, J, U)
 
   h = MPolyQuoLocalizedRingHom(W, V, [x//(y-1), y//(x-5)])
-  @test preimage(h, ideal(V, [x*(x-1), y*(y-3)])) == ideal(W, [x, y])
+  @test preimage(h, ideal(localized_ring(V), [x*(x-1), y*(y-3)])) == ideal(localized_ring(W), [x, y])
```

Comment:
This change has to be reverted. The `localized_ring` has nothing to do with `h` since it is a map `h : W -> V`. Hence the ideals for the input are not compatible with the domain and codomain. 

At least, this is what crashes for me when checking out this branch. Removing this, we get some problems down the road, but they seem to be due to unfinished replacements of `modulus` and `localized_modulus`.